backend/src/main/resources/application.yaml

```diff
@@ -8,8 +8,9 @@ spring:
     password: api123
     url: jdbc:postgresql://localhost:${SR_DB_PORT:5432}/simple_report
     hikari:
-      maximum-pool-size: 20
-      max-lifetime: 900000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
+      maximum-pool-size: 50 # Maximum size that the pool is allowed to reach, including both idle and active connections. Effectively reflects number of concurrent database request threads. (Ex: A size of 50 represents a maximum capacity of 50 concurrent DB requests.)
```

Comment:
going from 20 to 50 seems a bit aggressive. we're definitely treating the symptom here.

Reply:
Yes, this is an aggressive jump. I provided a bit more context behind this in the background information at the top of this PR. As things stand, this represents the number of concurrent database operations we can maintain. This is proposed as a stopgap to get us through the holiday period, while we try to get a handle on the potential design considerations that may need to be addressed.

Remember, we will likely see a spike of activity immediately following the new year as schools rush to test their students before the start of the new semester, and other facilities handle return-to-work and post-travel cases. I want to ensure we're not having to come back and bump this up again in two weeks.